src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+  --hover-color: #00acdc;
+}
+
+.container {
+  display: flex;
+  height: 60px;
```

Comment:
Avoid fixing the container size unless it's a specific requirement. It's better to let the content determine the height to prevent overflow or accidental scroll bars.